addon/helpers/build-secondary-nav-links.js

```diff
@@ -19,6 +19,7 @@ export default Ember.Helper.extend({  // Helper defined using a class, so can in
     currentUser: Ember.inject.service(),
     compute(params) { // Helpers defined using a class need a compute function
         const currentService = params[0].toUpperCase();
+        const serviceSubmitLink = Ember.isEmpty(params[1]) ? serviceLinks.preprintsSubmit : params[1] + 'submit';
```

Comment:
I think constructing the URL (appending `submit`) should happen where [`addPreprintUrl` is generated](https://github.com/CenterForOpenScience/ember-osf-reviews/pull/50/files#diff-3f6c41472bff6110ba0da1b4db3c97d3R26) -- in general, names should be accurate.

...and speaking of names, I guess `submitPreprintUrl` might be a better name for that property/argument/variable, for consistency. But that's not very important.

Reply:
I think passing the url as it is now will allow future possible re-use in other parts of the code. That is why I thought of it as generic at least from client app point of view.

Reply:
I think you suggested addPreprintUrl in the other PR

Reply:
I did suggest `addPreprintUrl`, but my word is not dogma, and I am often wrong -- question everything.

Good point, passing in the base URL is more reusable, and it would let us point the Search link to the right discover page, too. Just need to make the name make sense, then -- `preprintServiceUrl`? `baseServiceUrl`? Something else?

Reply:
While we're here, would you mind doing the search/discover link the same way?